commec/config/io_parameters.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+"""
+Defines the `Input and Output Screen Parameters` class, and associated dataclasses.
+Contains information pertinant to commec screen, including input configuration,
+and input fasta / query validation and parsing.
+"""
+import os
+import glob
+import argparse
+from dataclasses import dataclass
+from typing import Optional
+
+from commec.config.query import Query
+
+@dataclass
+class ScreenConfiguration:
+    """
+    Namespace for optional input parameters for screening; provided by parsing arguments. 
+    All have default values, which are set here, and are reflected into argparse.
+    """
+    threads: int = 1
+    search_tool: str = "blastx"
```

Comment:
`protein_search_tool` or `aa_search_tool` (since we always use `blastn` for nt)?

Reply:
Yeap! I'll make the change.